[Origin trials article] Add downtime warning (#3576)
GoogleChrome/web.dev: src/site/content/en/blog/origin-trials/index.md

authors: 
  - samdutton
date: 2020-06-22

> updated: 2020-07-27

Author: ```suggestion
updated: 2020-08-14
```

> updated: 2020-08-14

Reviewer: ```suggestion
updated: 2020-08-17
```

## How to register for an origin trial

1. Choose an origin trial from the [list of active trials](https://developers.chrome.com/origintrials/#/trials/active).
1. Request a token by clicking the **Request** button and filling out the form.
1. Add the token to your web pages,
   using a `<meta>` tag or an `Origin-Trial` HTTP response header.
   -  As a meta tag in the &lt;head&gt; of each page served:   
      `<meta http-equiv="origin-trial" content="TOKEN_GOES_HERE">`
   -  As an HTTP header:  
      `Origin-Trial: TOKEN_GOES_HERE`
1. Try out the new feature.
1. Submit feedback. Do this through the origin trial site. This feedback is 
   not public and is available only to a limited group of people on the Chrome
   team. Each trial also provides a link for spontaneous community feedback.
   This typically points to the the feature on GitHub or some other public
   channel.
1. When your token expires, you will get an email with a link to renew it.

>    In order to do so, you are again asked to submit feedback.

Author: It's obligatory actually, but this way it sounds more polite.

   
{% Aside 'warning' %}
  Even if an API lands unchanged after a successful origin trial,
  there is a short period between the end of the origin trial and the date the implementation ships
  in the browser when the API will not be available.

>   This is by design.

Author: ```suggestion
  This is by design. If we were to avoid the mandatory total-breakage period,
  we would bias toward also avoiding breakages in the API surface,
  and the final shipping API would potentially be worse for it.
```

>   there is a short period of time, between the end of the origin trial and the date the implementation ships

Author: It's not fixed. I don't think we want to commit to anything, not even vaguely.

>   date where the implementation ships in the browser where the API will not be available.

Reviewer 2: Maybe '... a short period of time, between the end of the origin trial and the date the implementation ships in the browser, when the API will not be available.'

Author: Ça! Thanks.

>   Please note that even if an API eventually lands unchanged after a successful origin trial,
>   there is an expected short period of time between the end date of the origin trial and the
>   date where the implementation ships in the browser where the API will not be available.

Author: ```suggestion
  Please note that even if an API eventually lands unchanged after a successful origin trial,
  there is a short period of time, between the end of the origin trial and the date the implementation ships
  in the browser, when the API will not be available.'
```

>   This is by design.

Author: Yes, I have made this explicit now.

>   there is an expected short period of time between the <em>end date of the origin trial</em> and the
>   <em>date where the implementation ships in the browser</em> where the API will not be available.

Author: ```suggestion
  there is an expected short period of time between the end date of the origin trial and the
  date where the implementation ships in the browser where the API will not be available.
```
